ivy/array/elementwise.py

```diff
@@ -1156,8 +1156,8 @@ def log(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
         >>> print(y)
         ivy.array([nan, nan, -inf, 0., 1.61, inf])
 
-        >>> x = ivy.array([[float('nan'), 1, 5.0, float('+inf')],\
-                           [+0, -1.0, -5, float('-inf')]])
+        >>> x = ivy.array([[float('nan'), 1, 5.0, float('+inf')],
+        ...                   [+0, -1.0, -5, float('-inf')]])
```

Comment:
for e.g. this line has an incorrect indentation, the `[` on this line should align with the second `[` on the previous line.

Reply:
okay noted!